.github/workflows/gpu-tests.yaml

```diff
@@ -29,6 +31,39 @@ jobs:
                   retention-days: 5
             - run: echo "This job's status is ${{ job.status }}."
 
+    LULESH-tests:
+        runs-on: [self-hosted, linux, x64]
+        container: gcr.io/gem5-test/gcn-gpu:latest
+        needs: build-gem5
+        timeout-minutes: 480 # 8 hours
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                # Scheduled workflows run on the default branch by default. We
+                # therefore need to explicitly checkout the develop branch.
+                  ref: develop
+
+            - name: Download build/GCN3_X86/gem5.opt
+              uses: actions/download-artifact@v3
+              with:
+                  name: weekly-test-${{ github.run_number }}-attempt-${{ github.run_attempt }}-gem5-build-gcn3
+                  path: build/GCN3_X86
+                # `download-artifact` does not preserve permissions so we need to set
+                # them again.
+            - run: chmod u+x build/GCN3_X86/gem5.opt
+
+            - name: Obtain LULESH
+              working-directory: ${{ github.workspace }}/lulesh
+              # Obtains the latest LULESH compatible with this version of gem5 via
+              # gem5 Resources.
+              run: build/GCN3_X86/gem5.opt util/obtain-resource.py lulesh -p lulesh
+
+            - name: Run LULUESH tests
+              working-directory: ${{ github.workspace }}
+              run: |
+                  build/GCN3_X86/gem5.opt configs/example/apu_se.py -n3 --mem-size=8GB --reg-alloc-policy=dynamic --benchmark-root="lulesh" -c \
+                  lulesh 0.01 2
```

Comment:
Actually, now that I look at this, did you test this?  I am surprised line 65 is "lulesh 0.01 2" instead of "0.01 2" -- maybe this is something with the new YAML format I'm not getting?

Reply:
It looks right to me.  I am using the left/right view on github so the original "lulesh" (no parameters) command was moved to a new line along with adding the parameters "0.01 2".  In my opinion the other view is more confusing

Reply:
Weird, just must be my browser's view then.